samples/logs/enc/Makefile

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
A sample for logging seems like an overkill.
If logging is more feature rich (ie encrypted) then a sample adds value.